pkgs/applications/misc/st/default.nix

```diff
@@ -4,30 +4,28 @@
 with stdenv.lib;
 
 stdenv.mkDerivation rec {
-  name = "st-0.6";
+  name = "st-0.7";
   
   src = fetchurl {
     url = "http://dl.suckless.org/st/${name}.tar.gz";
-    sha256 = "0avsfc1qp8zvshsfjwwrkvk411jlqy58z225bsdhjkl1qc40qcc5";
+    sha256 = "f7870d906ccc988926eef2cc98950a99cc78725b685e934c422c03c1234e6000";
   };
-
-  inherit patches;
+  
+  patches = with self; [ ./st-fix-deletekey.patch ];
 
   configFile = optionalString (conf!=null) (writeText "config.def.h" conf);
   preBuild = optionalString (conf!=null) "cp ${configFile} config.def.h";
   
   buildInputs = [ pkgconfig libX11 ncurses libXext libXft fontconfig ];
 
-  NIX_LDFLAGS = "-lfontconfig";
-
   installPhase = ''
     TERMINFO=$out/share/terminfo make install PREFIX=$out
   '';
     
   meta = {
     homepage = http://st.suckless.org/;
     license = stdenv.lib.licenses.mit;
-    maintainers = with maintainers; [viric];
+    maintainers = with maintainers; [andsild];
```

Comment:
Why did you remove @viric? There can be multiple maintainers, have they told you they're no longer maintaining this?